third_party/cpuinfo/BUILD.bazel

```diff
@@ -158,6 +160,18 @@ cc_library(
 
 ############################# Build configurations #############################
 
+config_setting(
+    name = "linux_aarch64",
```

Comment:
This part has already been added in https://github.com/tensorflow/tensorflow/commit/4f395601839390b08da3323a0608a9f22d3d0ac3

Reply:
removed.